Cubical/Categories/Displayed/Base/More.agda

```diff
@@ -2,19 +2,23 @@
 --
 module Cubical.Categories.Displayed.Base.More where
```

Comment:
This file is getting way too big, I eventually got a circular dependency from trying to do everything in here. We should split some of these constructions into their own modules